end_to_end_tests/golden-record/my_test_api_client/models/a_model.py

```diff
@@ -45,23 +48,36 @@ def to_dict(self) -> Dict[str, Any]:
 
                 nested_list_of_enums.append(nested_list_of_enums_item)
 
+        some_dict = self.some_dict if self.some_dict else None
```

Comment:
This is probably a latent bug but it means empty dict would be erased. Getting rid of dicts in this version anyway so nbd